src/proxyWorker.js

```diff
@@ -4,27 +4,6 @@
  * SPDX-License-Identifier: MIT
  */
 
-if (typeof console == 'undefined') {
```

Comment:
I'm not sure what this code could have been for.. there are no environments that I know of that supports workers but don't have a console object.